src/mixins/IndividuMixin.js

```diff
@@ -1,21 +1,45 @@
 import Individu from '@/lib/Individu'
 
+const autoValidatedSteps = [
+    'activite',
+    'nationalite',
+    'handicap',
+    'inapte_travail',
+    'garde_alterne',
+    'handicap',
+    'enfant_a_charge',
+    'taux_incapacite',
+    'ass_precondition_remplie',
+    'aah_restriction_substantielle_durable_acces_emploi',
+    'enfant_place'
+];
+
 export const createIndividuMixin = (props, optional) => {
     return {
         data: function() {
             const id = this.$route.params.id
             const role = id.split('_')[0]
             const { individu } = Individu.get(this.$store.getters.peopleParentsFirst, role, this.$route.params.id, this.$store.state.dates)
+            const autoValidate = autoValidatedSteps.includes(props)
             const value = individu[props]
             return {
                 error: false,
                 individu,
                 id,
                 value,
                 role,
+                autoValidate,
                 optional
             }
         },
+        watch: {
+           value() {
+                if (this.autoValidate) {
+                    this.onSubmit()
+                    // setTimeout(this.onSubmit, 400)
```

Comment:
Est-ce-que `setTimeout(this.onSubmit, 0)` fonctionne ?
On pourrait avoir un `setTimeout(this.onSubmit, process.env.VUE_APP_VALIDATION_DELAY || 0)` ? :sweat_smile: